bin/yk-config

```diff
@@ -21,10 +21,11 @@ OUTPUT=""
 usage() {
     echo "Generate C compiler flags for building against the yk JIT.\n"
     echo "Usage:"
-    echo "    yk-config <mode> <--cc|--cxx|--ar|--ranlib|--cppflags|--cflags|--ldflags>\n"
+    echo "    yk-config <mode> [--prelink <passes>] [--postlink <passes>] <--cc|--cxx|--ar|--ranlib|--cppflags|--cflags|--ldflags>\n"
```

Comment:
Great!

One tiny request: can we rename `--prelink` and `--postlink` to `--prelink-pipeline` and `--postlink-pipeline`?

Cheers.